ckanext/ontario_theme/plugin.py

```diff
@@ -962,6 +972,9 @@ def update_config_schema(self, schema):
         remove_whitespace = toolkit.get_converter('remove_whitespace')
 
         schema.update({
+            'ckanext.ontario_theme.site_title_fr':          [ignore_missing,
+                                                             ignore_not_sysadmin,
+                                                             unicode_safe],
```

Comment:
what's the purpose of ignore_missing, ignore_not_sysadmin, and unicode_safe?

Reply:
Ignore missing means config will be updated with or without the FR title, it’s not required.
Ignore not sysadmin means non sysadmins cannot change this field. 
Unicode safe means it will try to convert input to Unicode. Accents appear as Unicode in the source for example.